docs/ai_actions/install_ai_actions.md

```diff
@@ -105,10 +105,25 @@ If specific users from the team are supposed to modify scenario settings, you mu
 ### Create token
 
 Navigate to your [[= product_name_connect =]] user's profile, and on the **API ACCESS** tab, create a new token.
-Copy the token code that appears next to the label.
+Select the following scopes to set permissions needed to enable the integration of platforms:
+
+- `custom-property-structures:read`
+- `custom-property-structures:write`
+- `scenarios:read`
+- `scenarios:write`
+- `hooks:read`
+- `hooks:write`
+- `templates:read`
+- `templates:write`
+- `udts:read`
+- `udts:write`
+- `team-variables:read`
+- `team-variables:write`
```

Comment:
Is there a reason for this order?
It's not alphabetical.
It doesn't match the screenshot.